--- a/ipv6calc/Makefile.in	2024-02-15 12:55:16.000000000 +0100
+++ b/ipv6calc/Makefile.in	2024-05-02 11:31:49.219728197 +0200
@@ -15,7 +15,7 @@
 CFLAGS += @CFLAGS_EXTRA@
 
 LDFLAGS += @LDFLAGS@
-LDFLAGS += @LDFLAGS_EXTRA@
+LDFLAGS_EXTRA = @LDFLAGS_EXTRA@
 
 INCLUDES= @MD5_INCLUDE@ @GETOPT_INCLUDE@ @MMDB_INCLUDE_L1@ @IP2LOCATION_INCLUDE_L1@ -I../ -I../lib/ -I../databases/lib/
 
@@ -49,10 +49,10 @@
 		cd ../ && ${MAKE} lib-make
 
 ipv6calc:	$(OBJS) libipv6calc libipv6calc_db_wrapper
-		$(CC) -o ipv6calc $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) -lm
+		$(CC) -o ipv6calc $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) -lm
 
 static:		ipv6calc
-		$(CC) -o ipv6calc-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA_STATIC) -lm -static
+		$(CC) -o ipv6calc-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) $(LDFLAGS_EXTRA_STATIC) -lm -static
 
 distclean:
 		${MAKE} clean
--- a/ipv6logstats/Makefile.in	2024-02-15 12:55:16.000000000 +0100
+++ b/ipv6logstats/Makefile.in	2024-05-02 11:32:47.227751047 +0200
@@ -15,7 +15,7 @@
 CFLAGS += @CFLAGS_EXTRA@
 
 LDFLAGS += @LDFLAGS@
-LDFLAGS += @LDFLAGS_EXTRA@
+LDFLAGS_EXTRA = @LDFLAGS_EXTRA@
 
 INCLUDES= $(COPTS) @MD5_INCLUDE@ @GETOPT_INCLUDE@ @MMDB_INCLUDE_L1@ @IP2LOCATION_INCLUDE_L1@ -I../ -I../lib/ -I../databases/lib/
 
@@ -47,10 +47,10 @@
 $(OBJS):	ipv6logstatsoptions.h ipv6logstatshelp.h ipv6logstats.h
 
 ipv6logstats:	$(OBJS) libipv6calc.a libipv6calc_db_wrapper.a
-		$(CC) -o ipv6logstats $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) -lm
+		$(CC) -o ipv6logstats $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) -lm
 
 static:		ipv6logstats
-		$(CC) -o ipv6logstats-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA_STATIC) -lm -static
+		$(CC) -o ipv6logstats-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) $(LDFLAGS_EXTRA_STATIC) -lm -static
 
 distclean:
 		${MAKE} clean
--- a/ipv6logconv/Makefile.in	2024-02-15 12:55:16.000000000 +0100
+++ b/ipv6logconv/Makefile.in	2024-05-02 11:33:30.096767903 +0200
@@ -15,7 +15,7 @@
 CFLAGS += @CFLAGS_EXTRA@
 
 LDFLAGS += @LDFLAGS@
-LDFLAGS += @LDFLAGS_EXTRA@
+LDFLAGS_EXTRA = @LDFLAGS_EXTRA@
 
 INCLUDES= @MD5_INCLUDE@ @GETOPT_INCLUDE@ @IP2LOCATION_INCLUDE_L1@ @MMDB_INCLUDE_L1@ -I../ -I../lib/ -I../databases/lib/
 
@@ -47,10 +47,10 @@
 		cd ../ && ${MAKE} lib-make
 
 ipv6logconv:	$(OBJS) libipv6calc libipv6calc_db_wrapper
-		$(CC) -o ipv6logconv $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) -lm
+		$(CC) -o ipv6logconv $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) -lm
 
 static:		ipv6logconv
-		$(CC) -o ipv6logconv-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA_STATIC) -lm -static
+		$(CC) -o ipv6logconv-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) $(LDFLAGS_EXTRA_STATIC) -lm -static
 
 distclean:
 		${MAKE} clean
--- a/ipv6loganon/Makefile.in	2024-02-15 12:55:16.000000000 +0100
+++ b/ipv6loganon/Makefile.in	2024-05-02 11:34:15.561785762 +0200
@@ -15,7 +15,7 @@
 CFLAGS += @CFLAGS_EXTRA@
 
 LDFLAGS += @LDFLAGS@
-LDFLAGS += @LDFLAGS_EXTRA@
+LDFLAGS_EXTRA = @LDFLAGS_EXTRA@
 
 INCLUDES= $(COPTS) @MD5_INCLUDE@ @GETOPT_INCLUDE@ @IP2LOCATION_INCLUDE_L1@ @MMDB_INCLUDE_L1@ -I../ -I../lib/
 
@@ -47,10 +47,10 @@
 		cd ../ && ${MAKE} lib-make
 
 ipv6loganon:	$(OBJS) libipv6calc libipv6calc_db_wrapper
-		$(CC) -o ipv6loganon $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) -lm
+		$(CC) -o ipv6loganon $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) -lm
 
 static:		ipv6loganon
-		$(CC) -o ipv6loganon-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA_STATIC) -lm -static
+		$(CC) -o ipv6loganon-static $(OBJS) $(GETOBJS) $(LDFLAGS) $(LIBS) $(LDFLAGS_EXTRA) $(LDFLAGS_EXTRA_STATIC) -lm -static
 
 distclean:
 		${MAKE} clean
