diff --git a/configure b/configure
index f6ff2d0..31f4244 100755
--- a/configure
+++ b/configure
@@ -399,7 +399,7 @@ __END__
 #take_prog	Y
 
 #file		/usr/bin/file	# it is better to let configure do the checks
-HILITE		Y		# we do want syntax highlighting for --fixed
+HILITE		N		# we do want syntax highlighting for --fixed
 LESS_ADVANCED_PREPROCESSOR N	# always interpret textlike files (html, ...)
 
 # compression programs
@@ -407,10 +407,10 @@ bzip2		Y	# include bzip2 and unzip code, it is fairly essential
 unzip		Y
 rar		Y
 unrar		Y
-lzip		N	# default for lzma, lzip, xz, 7za should probably be N
+lzip		y	# default for lzma, lzip, xz, 7za should probably be N
 lzma		Y
 xz		Y
-7za		Y
+7za		N
 7zr		N	# included in 7za code
 # essential utilities
 cpio		Y
@@ -419,18 +419,18 @@ groff		Y
 html2text	Y	# rarely found but is a good alternative to elinks
 iconv		Y
 links		Y
-lynx		Y
+lynx		N
 nm		Y
-rpm		Y
+rpm		N
 rpmunpack	N	# fallback for rpm2cpio only, code inserted anyway
-rpm2cpio	Y
+rpm2cpio	N
 w3m		Y
 # code for these programs should be included in lesspipe.sh
 antiword	Y	# one of antiword or catdoc should be installed
 catdoc		Y
 dvi2tty		Y	# nice to have
 pstotext	Y	# pstotext is better than the fallback ps2ascii
-ps2ascii	N	# ps2ascii code is already in the pstotext part
+ps2ascii	Y	# ps2ascii code is already in the pstotext part
 iconv		Y
 gpg		Y
 pdftohtml	Y
@@ -451,7 +451,7 @@ ppthtml		Y
 unrtf		Y
 xlhtml		Y
 # for specific operating systems
-dpkg		Y	# for debian systems, fallback is tar+g(un)zip
+dpkg		N	# for debian systems, fallback is tar+g(un)zip
 ### please change to Y on MacOSX
 lsbom		N	# we are not on MacOSX
 plutil		N	# we are not on MacOSX
