diff --git a/lesspipe.sh.in b/lesspipe.sh.in
index a593617..7c311bb 100755
--- a/lesspipe.sh.in
+++ b/lesspipe.sh.in
@@ -532,7 +532,7 @@ parsehtml () {
 #ifdef links
   elif cmd_exist links; then
     if [[ "$1" = - ]]; then set - -stdin; fi
-    links -dump -force_html "$1"
+    links -dump -force-html "$1"
 #endif
   fi
 }
diff --git a/lesspipe.sh b/lesspipe.sh
index b7d50bd..0ea0598 100755
--- a/lesspipe.sh
+++ b/lesspipe.sh
@@ -463,7 +463,7 @@ parsehtml () {
     nodash "elinks -dump -force-html" "$1"
   elif cmd_exist links; then
     if [[ "$1" = - ]]; then set - -stdin; fi
-    links -dump -force_html "$1"
+    links -dump -force-html "$1"
   fi
 }
 
