--- files/79-onerng.rules.orig	2017-01-15 03:49:59.000000000 +0100
+++ files/79-onerng.rules	2021-01-26 17:18:55.260131322 +0100
@@ -1,6 +1,2 @@
-#
-#	start rngd  for the onerng entropy generator
-#
-ACTION=="add", ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6086", ENV{ID_MM_DEVICE_IGNORE}="1", ENV{ID_MM_CANDIDATE}="0", OWNER:="root", GROUP:="root", MODE:="0600", RUN+="/sbin/onerng.sh %k"
-ACTION=="remove", ENV{ID_VENDOR_ID}=="1d50", ENV{ID_MODEL_ID}=="6086", RUN+="/sbin/onerng.sh kill %k"
-
+# Handle the onerng entropy generator
+ACTION=="add", ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6086", ENV{ID_MM_DEVICE_IGNORE}="1", ENV{ID_MM_CANDIDATE}="0", OWNER:="root", GROUP:="root", MODE:="0600", SYMLINK+="onerng", RUN+="/usr/bin/sv up rngd-onerng", ENV{REMOVE_CMD}="/usr/bin/sv down rngd-onerng"
