--- a/drivers/bluetooth/btusb.c	2025-01-09 13:33:55.000000000 +0100
+++ b/drivers/bluetooth/btusb.c	2025-01-11 19:51:56.003356351 +0100
@@ -547,6 +547,9 @@
 	{ USB_VENDOR_AND_INTERFACE_INFO(0x0bda, 0xe0, 0x01, 0x01),
 	  .driver_info = BTUSB_REALTEK },
 
+	/* Alfa AWUS036AXML (MT7961) needs to be ignored */
+	{ USB_DEVICE(0x0e8d, 0x7961), .driver_info = BTUSB_IGNORE },
+
 	/* MediaTek Bluetooth devices */
 	{ USB_VENDOR_AND_INTERFACE_INFO(0x0e8d, 0xe0, 0x01, 0x01),
 	  .driver_info = BTUSB_MEDIATEK |
