--- a/net/ipv6/sit.c	2015-12-22 23:16:28.000000000 +0100
+++ b/net/ipv6/sit.c	2016-01-09 18:51:59.121607712 +0100
@@ -644,7 +644,8 @@
 
 	ipv6h = ipv6_hdr(skb);
 
-	if (unlikely(is_spoofed_6rd(tunnel, iph->saddr, &ipv6h->saddr))) {
+	if (unlikely(iph->saddr != /* 192.88.99.1 */  htonl(0xC0586301) &&
+		     is_spoofed_6rd(tunnel, iph->saddr, &ipv6h->saddr))) {
 		net_warn_ratelimited("Src spoofed %pI4/%pI6c -> %pI4/%pI6c\n",
 				     &iph->saddr, &ipv6h->saddr,
 				     &iph->daddr, &ipv6h->daddr);
